scripts/gas-refund-program/transactions-validation/validateTransactions.ts

```diff
@@ -38,12 +38,20 @@ export async function validateTransactions() {
   const guardian = GRPBudgetGuardian.getInstance();
 
   const lastEpochRefunded = await fetchLastEpochRefunded();
-  const startEpochForTxValidation = !lastEpochRefunded
+
+  const firstEpochOfYear = !lastEpochRefunded
     ? GasRefundGenesisEpoch
+    : Math.max(
+        GasRefundGenesisEpoch,
+        lastEpochRefunded - (lastEpochRefunded % TOTAL_EPOCHS_IN_YEAR),
+      );
```

Comment:
what's your intention here? The ternary's second part will result in a decimal epoch. perhaps the modulus of `(lastEpochRefunded % TOTAL_EPOCHS_IN_YEAR)` should be multiplied with epochs_per_year to get the number of epochs past the first? something like:
`lastEpochRefunded - ((lastEpochRefunded % TOTAL_EPOCHS_IN_YEAR) * TOTAL_EPOCHS_IN_YEAR),`

Reply:
as the variable says, it gets the first epoch of the year. How is that decimals ?

Reply:
There was a problem actually, but on different matter.
I was assuming that: 
- 1st epoch of 1st year  -> epoch 9 
- 1st epoch of 2nd year -> epoch 26 (matching `TOTAL_EPOCHS_IN_YEAR`)
- 1st epoch of 3rd year  -> epoch 52 (matching `TOTAL_EPOCHS_IN_YEAR*2`)

while it should be
- 9
- 35 (9+26)
- 61 (...)

Reply:
ecbef98fc1187b9f0a331f4f77f82831d48eb656